### ISSUE TITLE
Undirected multigraph edges are filled polygons in Matplotlib

### ISSUE THREAD

**Issue by vnmabus** [igraph/python-igraph#731]
**Describe the bug**
Plotting undirected multigraphs with the Matplotlib backend, the resulting image has filled edges.

**To reproduce**
Use the following code:

```python
import igraph as ig
import matplotlib.pyplot as plt

g = ig.Graph([(0,1), (1,0), (1,2), (2,0)], directed=False)

fig, ax = plt.subplots()
ig.plot(g, target=ax)
```

This generates the following image:
![graph](https://github.com/igraph/python-igraph/assets/2364173/4f214a11-5f41-4954-b244-b1c3f2bd3d5f)

**Version information**
I am using version 0.11.2 installed using pip.

**Comment by iosonofabio** [comment]
Thank you @vnmabus 

I thought I had caught all cases but evidently I missed one. I'll make a PR over the weekend.

**Comment by iosonofabio** [comment]
I made a quick PR for this, it's under control and apologies for the incorrect behaviour.

If this passes CI and is merged quickly, we could consider a hotfix minor release, depending on @ntamas 's availability over the next few days.

**Comment by szhorvat** [comment]
I'd like a C release quite soon (one more PR to merge). Let's do the two releases together

**Comment by ntamas** [comment]
This is now fixed in 0.11.3, which will be on PyPI in a few hours or so.
